backend/app.js

```diff
@@ -36,26 +38,77 @@ async function getId() {
 
 async function getInventory(idList, json) {
   let list = [];
-  for (let i = 0; i < json.data.length; i++) {
-    idList.push(json.data[i].relationships.options.data[0].id);
+
+  //this singleProdInfos creates and array for all the data of single type products
+  var singleProdInfos = json.data.filter(
+    item => item.relationships.options.data.length == 1
+  );
+
+  var singleProductIDAndName = [];
+  //this singleProductIDAndName creates an array that contains all the Products' IDs and Names
+  for (let i = 0; i < singleProdInfos.length; i++) {
+    singleProductIDAndName.push([
+      singleProdInfos[i].relationships.options.data[0].id,
+      singleProdInfos[i].attributes.name
+    ]);
   }
+
+  /*this includedList creates an array all of the product IDs in included.json and we use 
+  that to match with the IDs in singleProductIDAndName array and from there create a new 
+  nested arrays that contains the product name, quantity, price, and how many are sold.*/
```

Comment:
fix formatting of multiline comments, see your previous PR

Reply:
fixed